packages/rest-typings/src/v1/omnichannel.ts

```diff
@@ -1254,6 +1255,56 @@ const POSTOmnichannelContactsSchema = {
 
 export const isPOSTOmnichannelContactsProps = ajv.compile<POSTOmnichannelContactsProps>(POSTOmnichannelContactsSchema);
 
+type POSTUpdateOmnichannelContactsProps = {
+	contactId: string;
+	name?: string;
+	emails?: string[];
+	phones?: string[];
+	customFields?: Record<string, unknown>;
+	contactManager?: string;
+};
+
+const POSTUpdateOmnichannelContactsSchema = {
+	type: 'object',
+	properties: {
+		contactId: {
+			type: 'string',
+		},
+		name: {
+			type: 'string',
+			nullable: true,
```

Comment:
is name actually nullable? or it's enough with it not being present (so not in the `required` array)?

Reply:
it is an update endpoint, so besides the contactId, any field is optional to be updated, make sense?

Reply:
Yeah, but my point is: `nullable` allows you to pass `null` in here. Do you want that, or do you want this prop to just not be there? (not required and without nullable)
https://ajv.js.org/json-schema.html#nullable

Reply:
Oooh, I got it! thanks for the heads-up